### DISCUSSION TITLE
[LIKELY BUG] Starred repos don't receive username change

### DISCUSSION THREAD

**Discussion by vilewired** [community/community#113433] [Mobile]
I just tried to go through my starred repositories on the mobile app and it didn't find any. On the top I checked the name and it was my old username (I changed it). I resolved the issue by deleting the app storage and signing in again. I don't fully know if that's a bug or if that's just me experiencing that issue, but I wanted to share it anyway.

**Comment by esahmed21** [comment]
Thank you @vilev0  for reaching out and sharing your experience! Could you please confirm if you encountered this issue on the iOS or Android version of the mobile app? Your feedback is greatly appreciated.

**Reply by vilewired** [reply]
On the android version :)

**Reply by esahmed21** [reply]
Thank you for bringing this to our attention! I'll create an internal ticket for our team to investigate and address this matter.
